.github/workflows/syntax_check_cloud.yml

```
@@ -25,9 +25,23 @@ jobs:
     - name: Check old style classes
       if: ${{ !cancelled() }}
       # Only use parenthesis in class definition if there is inheritance (code-style)
-      run: |        
+      run: |
         grep -IrE --colour "class .+\(\).*:" --include=*.py ./src/odemis || exit 0
         if [[ $? -eq 0 ]]; then
            echo "The above syntax errors were found, only use parenthesis in class definition if there is inheritance (code-style)."
            exit 1  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)